### ISSUE TITLE
Only returns first 1,000 objects from S3 - Pydio Cells 4.0.0 RC3

### ISSUE THREAD

**Issue by wpuckering** [pydio/cells#425]
I created an S3-compatible data source (Wasabi), using “Keep data structure on storage”, and during sync only the first 1,000 files are detected (plus one additional node, probably the root). Re-sync doesn’t retrieve any more files than this, and my bucket has thousands.

This was not a problem in Pydio Cells 4.0.0 RC2. Thought I should bring this up since for a Release Candidate this is a pretty serious problem.

**Comment by bsinou** [comment]
> This was not a problem in Pydio Cells 4.0.0 RC2. Thought I should bring this up since for a Release Candidate this is a pretty serious problem.
Indeed. Thanks a lot for the testing and the feedback. We will definitively have a look at this before RC4.

**Comment by cdujeu** [comment]
Hey @wpuckering
this should be fixed in last commit, can you give it a try ? If you don't build by yourself, use https://download.pydio.com/pub/cells/dev/linux-amd64/

cheers

**Comment by cdujeu** [comment]
Should be OK in RC5.
